bindings/python/dnet_client

```diff
@@ -383,9 +383,11 @@ def process_lookup():
                 ('json_offset', r.record_info.json_offset),
                 ('json_size', r.record_info.json_size),
                 ('json_capacity', r.record_info.json_capacity),
+                ('json_checksum', r.record_info.json_checksum if r.record_info.json_checksum else None),
```

Comment:
`r.record_info.json_checksum` should be `None` by itself, shouldn't it?
```suggestion
                ('json_checksum', r.record_info.json_checksum or None),
```

Reply:
Is I understand, @sabramkin replaces empty string with `None`.
If not just `('json_checksum', r.record_info.json_checksum),` is better.

Reply:
Need a time to think and rework. This code isn't good, because no-checksum represented as `str` of length 64, filled by zeroes. So `None` never occurs.

Reply:
And I doubt whether uchar[64] is converted by boos-python as binary, I must research it.